tests/integration/controller/mandatorymodules/mandatory_modules_controller_test.go

```diff
@@ -0,0 +1,151 @@
+package mandatory_test
+
+import (
+	"context"
+	"errors"
+	"time"
+
+	"github.com/kyma-project/lifecycle-manager/api/shared"
+	"github.com/kyma-project/lifecycle-manager/api/v1beta2"
+	compdescv2 "github.com/open-component-model/ocm/pkg/contexts/ocm/compdesc/versions/v2"
+	apimetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8slabels "k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kyma-project/lifecycle-manager/pkg/status"
+	"github.com/kyma-project/lifecycle-manager/pkg/testutils/builder"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	. "github.com/kyma-project/lifecycle-manager/pkg/testutils"
+)
+
+var (
+	ErrWrongModulesStatus  = errors.New("modules status not correct")
+	ErrNoMandatoryManifest = errors.New("manifest for mandatory Module not found")
+)
+
+var _ = Describe("Mandatory Module Installation", Ordered, func() {
+	Context("Given Kyma with no Module and one mandatory ModuleTemplate on Control-Plane", func() {
+		kyma := NewTestKyma("no-module-kyma")
+		registerControlPlaneLifecycleForKyma(kyma)
+
+		It("Then Kyma CR should result in a ready state immediately as there are no modules", func() {
+			Eventually(KymaIsInState, Timeout, Interval).
```

Comment:
Let's introduce global timeout and interval to suite_test.go, like what we did in [e2e test](https://github.com/kyma-project/lifecycle-manager/blob/8b74fb092d3013565b6c10a5c297092aeda7fa57/tests/e2e/suite_test.go#L91-L94), and don't continue doing this individual configure to new test.

Reply:
Implemented